### PR TITLE
[DE] Translation backcover

### PR DESCRIPTION
**Pull request by DerSchlupf**
german Translation back-cover section:
image

### CODE REVIEW

File: translations/back_cover.tex/de.po [hunk 1]
```diff
@@ -82,3 +110,19 @@ msgid ""
 "\\svg{building_materials} \\svgeven[7]{arrow_right_gray} 1 "
 "\\svg{valuablegreater}} & \\lightcell{--} \\\\ \\end{tabularx} }"
 msgstr ""
+"\\hommtable[]{14}{ \\centering \\textbf{\\pagetarget{Trade Table}{Übersicht "
+"Handelskurse}}\\\\ \\bigskip \\begin{tabularx}{\\linewidth}{XXXX} "
+"\\darkcell{\\color{white}Verkaufe/ Erhalte} & \\darkcell{… und erhalte "
+"\\svg{gold}} & \\darkcell{… und erhalte \\svg{valuablegreater}} & "
+"\\darkcell{… und erhalte \\svg{building_materials}} \\\\ \\darkcell{Ich "
```
Comment (qwrtln) [inline translations/back_cover.tex/de.po:117]:
```suggestion
"\\darkcell{\\color{white}Verkaufe/ Erhalte} & \\darkcell{... und erhalte "
"\\svg{gold}} & \\darkcell{... und erhalte \\svg{valuablegreater}} & "
"\\darkcell{... und erhalte \\svg{building_materials}} \\\\ \\darkcell{Ich "
```

Comment (DarthGandalf) [reply translations/back_cover.tex/de.po:117]: What's wrong with …?

Comment (qwrtln) [reply translations/back_cover.tex/de.po:117]: Consistency. The table looks very asymmetric now.

**Review by qwrtln — APPROVED** [review]

**Comment by DerSchlupf** [comment]
Picture-Update above! Do you think, it's ready?

**Comment by qwrtln** [comment]
> Picture-Update above! Do you think, it's ready?

It is, merging.

**Pull request merged**